testbed/windows/Runner.sln

```diff
@@ -10,6 +10,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "Runner", "Runner.vcxproj",
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "ExamplePlugin", "..\..\plugins\example_plugin\windows\ExamplePlugin.vcxproj", "{95E21B2C-C18A-4CED-8509-585CB2570FDE}"
 EndProject
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "url_launcher_fde", "..\..\plugins\flutter_plugins\url_launcher_fde\windows\url_launcher_fde.vcxproj", "{18F565CD-7BD5-459C-9FEB-2E2379E88105}"

```

Comment:
Is having a duplicate UUID valid in VS? It might be a good idea to hand-edit the duplicated project to change the UUID.

Reply:
good eye!

Reply:
After looking more I see that the one on the left isn't the ID of the referenced project, and doesn't seem to be unique already (the line just outside of context uses the same one for the app project too). So it seems like this is not an issue.